client/main/ribbon.js

```diff
@@ -125,8 +125,14 @@ const RibbonView = Backbone.View.extend({
         this.$tabs = this.$header.find('.jmv-ribbon-tab');
 
         this.appMenu = new AppMenu({ el: this.$appMenu, model: this.model });
-        this.appMenu.on('shown', event => this._menuShown(event));
-        this.appMenu.on('hidden', event => this._closeMenus());
+        this.appMenu.on('shown', event => {
+            this._menuShown(event);
+            $('#header-buttons').hide();
+        });
+        this.appMenu.on('hidden', event => {
+            this._closeMenus();
+            $('#header-buttons').show();
```

Comment:
can you assign these to a member variable in `initialize()`? and then refer to them `this.$headerButtons...`

good practice to avoid querying the DOM over and over

`header-buttons` is probably too generic to use as an `id` too.